data/statuses.js

```diff
@@ -523,7 +523,8 @@ let BattleStatuses = {
 		onTryMove: function (target, source, effect) {
```

Comment:
```suggestion
		onTryMove: function (attacker, defender, effect) {
```

Reply:
idk what's wrong with the indentation here, you might need to accept this change manually.